jekyll/_cci2/deployment-integrations.md

```diff
@@ -142,137 +142,50 @@ To deploy to Azure, use a similar job to the above example that uses an appropri
 
 ## Heroku
 
-The built-in Heroku integration through the CircleCI UI is not implemented for CircleCI 2.0.
-However, it is possible to deploy to Heroku manually.
-
-First, create a bash script to set up Heroku and place the file in the `.circleci` directory:
-
- ```bash
- #!/bin/bash
- wget https://cli-assets.heroku.com/branches/stable/heroku-linux-amd64.tar.gz
- sudo mkdir -p /usr/local/lib /usr/local/bin
- sudo tar -xvzf heroku-linux-amd64.tar.gz -C /usr/local/lib
- sudo ln -s /usr/local/lib/heroku/bin/heroku /usr/local/bin/heroku
-
- cat > ~/.netrc << EOF
- machine api.heroku.com
-   login $HEROKU_LOGIN
-   password $HEROKU_API_KEY
- EOF
-
- cat >> ~/.ssh/config << EOF
- VerifyHostKeyDNS yes
- StrictHostKeyChecking no
- EOF
- ```
-
-**Note:** `sudo` is only necessary if the script runs in a Docker container with a non-root user.
-If the user _is_ root,
-remove `sudo` as it may not be installed.
-
-Next, install and authorize Heroku for the CircleCI account that owns the project.
-[Add environment variables]({{ site.baseurl }}/2.0/env-vars/) for the Heroku API key and login email to the CircleCI application as shown in the following image:
+[Heroku](https://www.heroku.com/) is a popular platform
+for hosting applications in the cloud.
+To configure CircleCI
+to deploy your application to Heroku,
+follow the steps below.
 
-![Add Environment Variables]({{ site.baseurl }}/assets/img/docs/walkthrough5.png)
+1. Create a Heroku account
+and [follow the instructions](https://devcenter.heroku.com/start)
+to set up a project in your chosen language.
 
-To connect to the Heroku Git server from CircleCI,
-create a new SSH key without a passphrase.
-The private key is added through the SSH Permissions page
-with a hostname of `git.heroku.com` as shown in the following image:
+2. Add the name of your Heroku application and your Heroku API key as [environment variables]({{ site.baseurl }}/2.0/env-vars/#adding-project-level-environment-variables).
+In this example, these variables are defined as `HEROKU_API_KEY` and `HEROKU_APP_NAME`, respectively.
 
-![Add SSH Key]({{ site.baseurl }}/assets/img/docs/walkthrough6.png)
+3. [Create a new SSH key](https://devcenter.heroku.com/articles/keys) without a passphrase,
+and [add the key to Heroku](https://devcenter.heroku.com/articles/keys#adding-keys-to-heroku).
 
-Note the private key's fingerprint for later reference.
-Add the public key to Heroku on the [Account page](https://dashboard.heroku.com/account).
+4. To connect to Heroku,
+[add the SSH key to CircleCI](https://circleci.com/docs/2.0/add-ssh-key/) with a **Hostname** of `git.heroku.com`.
 
-Finally, update your `config.yml` file with a job and workflow section similar to the following:
+5. In your `.circleci/config.yml`,
+create a `deploy-job`,
+`checkout` your code,
+and add a command
+to deploy the master branch to Heroku.
```

Comment:
Just to make this crystal-clear, I'd make this: `to deploy the master branch to Heroku via git.`